inclusive-design/wecount.inclusivedesign.ca
# Issue-144: Add unique page titles.

## Description

To resolve https://github.com/inclusive-design/wecount.inclusivedesign.ca/issues/144.

Add unique titles for each page.

## Steps to test

1. Visit each page;
2. Make sure they all have unique titles in the format of "{title} | We Count".
